pages/common/tlmgr-platform.md

```diff
@@ -0,0 +1,24 @@
+# tldr platform
+
+> Manage TeX Live platforms.
+> More information: <https://www.tug.org/texlive/tlmgr.html>.
+
+- List all available architectures in the package repository:
```

Comment:
Shouldn't it be `platform(s)` instead of `architecture(s)` everywhere?

Reply:
It's basically a synonym here. On one side, it's maybe a bit confusing, since the command is called `platform`, but I think architecture is just more fitting here.

Reply:
I think, it is **really** confusing to first `list [...] architectures` and then `switch to [...] platform`, so this **has** to be normalized. And I think that it has to be `platform`, because when you say `architecture`, you mean `platform`. They are not synonyms. An architecture is something like `amd64`, `aarch64`, `riscv64`, ... A platform is something like `x86_64-linux`, `universal-darwin`, ... and that is what you are talking about.